api/tests/src/util/test_text_extractor.py

```diff
@@ -65,3 +66,14 @@ def test_extract_pdf_text(fixture_file_path, fixture_file_path_val, expected) ->
     fixture_file_path = fixture_file_path(fixture_file_path_val)
     given = extract_text_from_file(fixture_file_path)
     assert given == expected
+
+
+def test_unprocessable_file_content(fixture_file_path):
+    with pytest.raises(UnprocessableFileContent, match="UnprocessableFileContent"):
+        extract_text_from_file(
+            fixture_file_path(f"{TEST_FILE_DIR}/bytes_named_as_text.txt"), raise_on_error=True
+        )
+        extract_text_from_file(
+            fixture_file_path(f"{TEST_FILE_DIR}/text_data_as_bytes_content.pdf"),
+            raise_on_error=True,
+        )
```

Comment:
If you're trying to test that both of these error, put them in separate `pytest.raises` blocks.